docs/user-guide/labels.md

```diff
@@ -126,38 +148,39 @@ LIST and WATCH operations may specify label selectors to filter the sets of obje
 
 Both label selector styles can be used to list or watch resources via a REST client. For example targetting `apiserver` with `kubectl` and using _equality-based_ one may write:
 
-```shell
+```console
 $ kubectl get pods -l environment=production,tier=frontend
 ```
 
 or using _set-based_ requirements:
 
-```
+```console
 $kubectl get pods -l 'environment in (production),tier in (frontend)'
 ```
 
-As already mentioned _set-based_ requirements are more expressive.  For instance, they can implement the _OR_ operator:
+As already mentioned _set-based_ requirements are more expressive.  For instance, they can implement the _OR_ operator on values:
 
-```shell
+```console
 $kubectl get pods -l 'environment in (production, qa)'
 ```
 
 or restricting negative matching via _exists_ operator:
 
-```shell
+```console
 $kubectl get pods -l 'environment,environment notin (frontend)'
 ```
 
-Kubernetes also supports two objects that use label selectors to keep track of their members, `service`s and `replicationcontroller`s:
+### Set references in API objects
+
+Some Kubernetes objects, such as [`service`s](services.md) and [`replicationcontroller`s](replication-controller.md), also use label selectors to specify sets of other resources, such as [pods](pods.md).
 
-* `service`: A [service](services.md) is a configuration unit for the proxies that run on every worker node.  It is named and points to one or more pods.
-* `replicationcontroller`: A [replication controller](replication-controller.md) ensures that a specified number of pod "replicas" are running at any one time.
+#### Service and ReplicationController
 
-The set of pods that a `service` targets is defined with a label selector. Similarly, the population of pods that a `replicationcontroller` is monitoring is also defined with a label selector. For management convenience and consistency, `services` and `replicationcontrollers` may themselves have labels and would generally carry the labels their corresponding pods have in common. Labels selectors for both objects are defined in `json` or `yaml` files using `map` and only _equality-based_ requirement selectors are supported:
+The set of pods that a `service` targets is defined with a label selector. Similarly, the population of pods that a `replicationcontroller` should monitor is also defined with a label selector. For management convenience and consistency, `services` and `replicationcontrollers` may themselves have labels and would generally carry the labels their corresponding pods have in common. Labels selectors for both objects are defined in `json` or `yaml` files using maps, and only _equality-based_ requirement selectors are supported:
```

Comment:
maybe s/monitor/manage/

"would generally carry" is pretty ambiguous; you could make it clearer that this is describing a "best practice" and not something the system does automatically